docs/NativeCodeGeneration.md

```diff
@@ -21,6 +21,15 @@ The verifier accepts an eBPF program compiled into an Executable Linux Format (E
 response depending on whether it can prove the program is safe. The verified ELF file can then be used to generate a PE
 image that can be loaded and used as an eBPF program.
 
+# Usage
+
+From the root of the eBPF-For-Windows project (from a VS Developer Command Prompt), after building the project, run:
+```
+powershell scripts\Convert-BpfToNative.ps1 -ProgramName my_program
```

Comment:
Minor: I think it would be more intuitive if the user passes the whole file name, and the script handles to remove the ".o" extension.